Weather App/viewModel/UserLocationViewModel.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  UserLocationViewModel.swift
+//  Weather App
+//
+//  Created by Piotr Chlapinski on 21/07/2022.
+//
+
+import Foundation
+import MapKit
+
+class UserLocationViewModel: NSObject, CLLocationManagerDelegate {
```

Comment:
what do you think to introduce protocols to each model to use only those things what you really need and help to write unit tests